install/overlays/ibmcloud/cri_runtime_endpoint.yaml

```diff
@@ -10,13 +10,16 @@ spec:
     spec:
       containers:
       - name: cloud-api-adaptor-con
+        env:
+        - name: CRI_RUNTIME_ENDPOINT
+          value: /run/containerd/containerd.sock
         volumeMounts:
-        - mountPath: /run/peerpod/cri-runtime.sock # in-container CRI_RUNTIME_ENDPOINT default
+        - mountPath: /run/containerd # in-container CRI_RUNTIME_ENDPOINT default
           name: cri-runtime-endpoint
```

Comment:
It would be good to have an explanation on why the Socket mount is not working when it's working for other cases ?

Reply:
Sure, I'll take some time to investigate it.

Reply:
Thanks a lot @genjuro214. Appreciate it.

Reply:
Did a quick search, not sure is that the case, https://stackoverflow.com/questions/53491603/kubernetes-volumemounts-a-file-not-a-directory

Reply:
I tried the operator against IKS cluster, looks work well without the change in `mountPath`, could it because you're using a kubeadm cluster in IBM Cloud? @genjuro214 , I'd suggest we drop the `mountPath` change in this PR and monitor it in IKS later.